src/components/Footer.tsx

```diff
@@ -1,7 +1,61 @@
+import Link from "next/link";
+import Image from "next/image";
+import { links } from "@/data/Footer";
+import logo from "@/public/Footer.webp";
+
 const Footer = () => {
   return (
-    <div className="flex flex-row bg-hlg-red-200 font-archivo-narrow text-2xl">
-      Footer
+    <div className="flex h-[375px] flex-row overflow-hidden bg-gradient-to-b from-black to-hlg-red-200 font-archivo-black text-2xl">
+      {/* Information Text White */}
+      <div className="z-50 ml-10 mt-[5%] flex flex-col space-y-3 text-3xl text-[#b5b5b5]">
+        <p>UCR</p>
+        <p className="pl-[25%]">HIGHLANDER</p>
+        <p className="pl-[100%]">GLOVES</p>
+      </div>
+      <div className="absolute z-50 m-auto ml-[66%] mt-[5%] flex flex-col text-right font-anton text-[#B5B5B5]">
+        <p className="text-5xl">CONTACT INFORMATION</p>
+        <p className="pt-[6%]">2060 Chicago Avenue A23</p>
+        <p className="pt-[3%]">Riverside, CA 92507</p>
+        <p className="pt-[3%]">United States</p>
+        <p className="pt-[3%]">email: highlandergloves14@gmail.com</p>
+      </div>
+
+      {/* Information Text Black */}
+      <div className="absolute left-[0.5%] z-20 ml-10 mt-[5.5%] flex flex-col space-y-3 text-3xl text-black">
+        <p>UCR</p>
+        <p className="pl-[25%]">HIGHLANDER</p>
+        <p className="pl-[100%]">GLOVES</p>
+      </div>
+      <div className="absolute z-20 m-auto ml-[66.5%] mt-[5.5%] flex flex-col text-right font-anton text-black">
+        <p className="text-5xl">CONTACT INFORMATION</p>
+        <p className="pt-[6%]">2060 Chicago Avenue A23</p>
+        <p className="pt-[3%]">Riverside, CA 92507</p>
```

Comment:
instead of absoluting the text for the layering, can you use shadow? reference the about us component (line 16).